app/src/main/java/com/vero/tasky/auth/presentation/login/LoginState.kt

```diff
@@ -0,0 +1,7 @@
+package com.vero.tasky.auth.presentation.login
+
+data class LoginState(
+    val isLoading: Boolean = false,
+    val isPasswordVisible: Boolean = false,
```

Comment:
What's the use of this one?

PasswordTextField has a state that already does this